.github/workflows/ci.yml

```diff
@@ -11,16 +11,11 @@ jobs:
     runs-on: ubuntu-latest
 
     steps:
-      - uses: actions/checkout@v3
+      - uses: purcell/setup-emacs@master
+        with:
+          version: 28.1
 
-      - name: Install project dependencies
-        env:
-          EMACS_VERSION: 28.1
-        run: |
-          export EMACS_DIR=$HOME/emacs
-          # Install Emacs 26 from upstream
-          bash -e <(curl -fsSkL 'https://raw.githubusercontent.com/vermiculus/emake.el/d8ab7790f5823547fd4e7aec2218771a3dda0638/build-emacs')
-          pushd . && cd $EMACS_DIR && sudo make install && popd
+      - uses: actions/checkout@v3
```

Comment:
Could you pin the action to a SHA instead of a tag? See https://exercism.org/docs/building/github/gha-best-practices#h-pin-actions-to-shas for the reasoning (I'm fine with this being updated in a follow-up PR).

Reply:
Sure, I pinned the latest commit on the master branch now because the last release was in 2020.

Reply:
Done!